core_java/twitter/.gitignore

```diff
@@ -0,0 +1,2 @@
+.idea
+.idea
```

Comment:
you should have only one .gitignore file under the project root directory.  remove this